JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2986,32 +2968,25 @@
   )
 }
 .mmMessageAddedTerms    <- function(terms, grouping) {
-  if (length(terms) < 2) {
-    return(
-      gettextf(
-        "Lower order random effects terms need to be specified in presence of the higher order random effects terms. Therefore, the following random effects term was added to the '%s' random effects grouping factor: '%s.'",
-        grouping,
-        paste0("'", terms, "'", collapse = ", ")
-      )
-    )
-  } else{
-    return(
-      gettextf(
-        "Lower order random effects terms need to be specified in presence of the higher order random effects terms. Therefore, the following random effects terms were added to the '%s' random effects grouping factor: '%s.'",
-        grouping,
-        paste0("'", terms, "'", collapse = ", ")
-      )
-    )
-  }
+  sprintf(
+    ngettext(
+      length(terms) < 2,
+      "Lower order random effects terms need to be specified in presence of the higher order random effects terms. Therefore, the following random effects term was added to the '%s' random effects grouping factor: '%s.'",
+      "Lower order random effects terms need to be specified in presence of the higher order random effects terms. Therefore, the following random effects terms were added to the '%s' random effects grouping factor: '%s.'"
+    ),
+    grouping,
+    paste0("'", terms, "'", collapse = ", ")
+  )
 }
 .mmMessageMissingRows   <- function(value) {
-  if (value < 2) {
-    return(gettextf("%i observation was removed due to missing values.",
-                    value))
-  } else{
-    return(gettextf("%i observations were removed due to missing values.",
-                    value))
-  }
+  sprintf(
+    ngettext(
+      value < 2,
```

Comment:
Unless you specifically want to group `0` and `1` together you should use `ngettext(length(value), "..", "..")` (so no boolean, but an integer as argument to `ngettext()`)

`0 observation was removed due to missing values.` is not correct.

This goes for quite a few of these changed lines

Reply:
the 's' is actually correct at the end - there might be more missing values within one observation. We do not check those.

These messages aren't called when the value == 0

Reply:
We've solved this over Slack, but:

`0 observation was removed` is correct not though :)
These are your sentences:
```
 "%i observation was removed due to missing values.",
 "%i observations were removed due to missing values."
```

Reply:
0 observation was removed cannot happen. The function is called only if `value > 0`

Reply:
I know, but the English is not correct. It was in response to this:

> the 's' is actually correct at the end - there might be more missing values within one observation. We do not check those.

Reply:
oh, my English is probably not good enough to understand the problem with my English :D